lib/ex_unit/lib/ex_unit/capture_log.ex

```diff
@@ -115,11 +117,11 @@ defmodule ExUnit.CaptureLog do
           {:DOWN, ^ref, :process, ^parent, _reason} -> :ok
           {:gen_event_EXIT, {Console, ^pid}, _reason} -> :ok
         end
-      {:EXIT, reason} ->
-        :proc_lib.init_ack({:error, reason})
-      {:error, reason} ->
-        :proc_lib.init_ack({:error, reason})
+      other ->
+        :proc_lib.init_ack(other)
```

Comment:
We still want some consistency with the error handling so we should still call`:proc_lib.init_ack({:error, reason_or_other})`. It's not essential to handle the `other` cases except to wrap in an error tuple when sending back to the parent. We want to do this because if the proxy crashes (e.g. we have an `exit/1` from `:gen_event`) without returning an error then `proc_lib.start` returns `{:error, exit_reason}`. Then all unexpected errors are of the form `{:error, exit_reason}`.